srv/components/provisioner/teardown/init.sls

```diff
@@ -15,10 +15,15 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
 
+include:
+  - components.provisioner.teardown.package_remove
+  - components.provisioner.teardown.remove_repos
+
+
 Delete rsyslog provisioner conf:
```

Comment:
don't you want to move that into rsyslog.sls or config.sls or something like that

Reply:
created rsyslog.sls